nomic/aws/sagemaker.py

```diff
@@ -0,0 +1,197 @@
+# Note: This file contains methods to prepare embedding requests for Sagemaker.
+# It may make sense to move code eventually to embed.py or somewhere more generic but
+# it currently lives here to separate out dependencies.
+
+import hashlib
+import logging
+from enum import Enum
+from pathlib import Path
+from typing import List, Optional
+
+import boto3
+import numpy as np
+import tritonclient.http.aio as aiohttpclient
```

Comment:
you want the non-asyncio version (just drop the `.aio` and the api is the same, sans-`await`)